src/bp/core/services/dialog/instruction/strategy.ts

```diff
@@ -128,14 +136,33 @@ export class ActionStrategy implements InstructionStrategy {
 
     args = _.mapValues(args, value => renderTemplate(value, actionArgs))
 
+    let actionServerId
+    if (chunks.length === 3) {
+      actionServerId = chunks[2]
+    }
```

Comment:
This logic will not always work, nothing prevents you from having a space in your arguments (which is just an object that's been stringified)

Reply:
Fixed in d574f8fd1e835d3c10514c9b86ec53e366eb2f1c